documentation.css

```diff
@@ -85,17 +99,23 @@ body {
 
 .component-navigation {
     height: 13rem;
+    width: 100%;
     box-shadow: 0 2px 3px -1px rgb(0 0 0 / 10%);
     display: flex;
     align-items: flex-end;
     justify-content: space-between;
-    flex-wrap: wrap;
+    /* flex-wrap: wrap; */
 }
 
 .navigation-pri-part {
     display: flex;
```

Comment:
The hamburger icon is not center-aligned. Please align it.

Reply:
Done with that